modules/preview-react/table/stories/examples/FixedColumn.tsx

```diff
@@ -75,8 +75,10 @@ export const FixedColumn = () => {
   ];
   return (
     <>
-      <Heading size="small">Performance Car Specs</Heading>
-      <Table>
+      <Heading size="small" marginBottom="s">
+        Performance Car Specs
+      </Heading>
+      <Table width={690}>
```

Comment:
I assigned a width to the `Table` in the Fixed Column example to guarantee the fixed column is triggered. I picked `690px` since the site is the primary destination for consuming these docs, and that width happens to look good on the site.

The site displays the example within an overflow container which prevents the fixed column from being triggered -- instead the container itself has a horizontal scroll for the entire table.

**Without width:**

https://user-images.githubusercontent.com/146020/234093199-39046210-2e74-4f04-a570-c01dc720dca2.mov

**With width:**

https://user-images.githubusercontent.com/146020/234093314-35a811d9-0068-4b8c-9111-74491554bddf.mov

Reply:
Good call. I wasn't aware of this when building the Table examples